apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -1,30 +1,34 @@
 /**
  * External dependencies
  */
-import { has, isEmpty, keyBy, map } from 'lodash';
-import { __ } from '@wordpress/i18n';
+import { isEmpty } from 'lodash';
+import { __, sprintf } from '@wordpress/i18n';
 import { compose } from '@wordpress/compose';
-import { Modal } from '@wordpress/components';
+import { Button, Modal } from '@wordpress/components';
 import { registerPlugin } from '@wordpress/plugins';
 import { withDispatch, withSelect } from '@wordpress/data';
-import { parse as parseBlocks } from '@wordpress/blocks';
 import { Component } from '@wordpress/element';
 import '@wordpress/nux';
 
 /**
  * Internal dependencies
  */
-import replacePlaceholders from './utils/replace-placeholders';
 import './styles/starter-page-templates-editor.scss';
 import TemplateSelectorControl from './components/template-selector-control';
+import TemplateSelectorPreview from './components/template-selector-preview';
 import { trackDismiss, trackSelection, trackView, initializeWithIdentity } from './utils/tracking';
 
 class PageTemplateModal extends Component {
 	state = {
 		isLoading: false,
+		previewBlocks: [],
+		slug: '',
+		title: '',
 	};
 
 	constructor( props ) {
+		// eslint-disable-next-line no-console
+		console.time( 'PageTemplateModal' );
```

Comment:
@retrofox Did we want to leave these console lines in?

Reply:
clearly, we don't.

Reply:
I wonder if we can keep these lines for a pair of days more since we still need to work on this code.